### DISCUSSION TITLE
Azure SQL Server support?

### DISCUSSION THREAD

**Discussion by juju4** [thinkst/canarytokens#243] [Feature Requests]
I did some testing on Azure SQL server and default canarytokens does not seem to be possible there mostly because of absence of extended stored procedures like xp_fileexist or xp_dirtree.

https://stackoverflow.com/questions/56521073/can-we-check-if-the-file-exists-or-not-in-azure-sql

raiserror/throw would be an alternate option through logs
Any other recommended options?

Thanks a lot for your work and sharing!

**Comment by jayjb** [comment]
Hi @juju4,

Currently I haven't looked into Azure SQL but it is an interesting thought. Ill bring up with the team and see what we can figure out.

Thanks for the recommendation.

**Comment by juju4** [comment]
Any update/new option?

Thanks
